src/functionalTest/java/uk/gov/hmcts/reform/pip/account/management/utils/FunctionalTestBase.java

```diff
@@ -107,6 +122,17 @@ protected Response doPutRequest(final String path, final Map<String, String> add
             .thenReturn();
     }
 
+    protected Response doPutRequestWithBody(final String path,
```

Comment:
This new method here looks identical to the one right below it. Can we just keep one of them?

Reply:
good spot :) I've removed this now